package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-konami",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
This is my first github PR so I'm unsure of etiquette.. I bumped the version number?  Should I not do that?